client/index.ejs

```diff
@@ -1,121 +1,163 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<% include ./partials/head %>
-		<link rel="stylesheet" href="https://openlayers.org/en/v5.3.0/css/ol.css" type="text/css">
-		<link rel="stylesheet" type="text/css" href="css/index.css">
-		<title>DAS</title>
-	</head>
-	<body>
-		<% include ./partials/header %>
+  <head>
+    <% include ./partials/head %>
+    <link
```

Comment:
For future reference, should something like this be split onto multiple lines?

Reply:
And on the topic of style, a lot of the .ejs files use tabs but should we be using 2 spaces?

Reply:
Perhaps the EJS stuff was done after we added prettier maybe? I actually like the 2 spaces since bootstrap does add a lot of elements...